src/handlers/comment/handlers/assign.ts

```diff
@@ -1,11 +1,12 @@
-import { addAssignees, addCommentToIssue, getCommentsOfIssue } from "../../../helpers";
+import { addAssignees, addCommentToIssue, getCommentsOfIssue, listIssuesForRepo } from "../../../helpers";
 import { getBotConfig, getBotContext, getLogger } from "../../../bindings";
 import { Payload, LabelItem, Comment, IssueType } from "../../../types";
 import { deadLinePrefix } from "../../shared";
 import { IssueCommentCommands } from "../commands";
 import { getWalletAddress } from "../../../adapters/supabase";
 
 export const assign = async (body: string) => {
+  let maxIssues = process.env.BOUNTY_HUNTER_MAX_OPENED_ISSUES;
```

Comment:
This parameter `BOUNTY_HUNTER_MAX_OPENED_ISSUES` is supposed to be read from the bot's yml config https://github.com/ubiquity/bounty-bot/blob/development/src/bindings/config.ts#L21

Example config file: https://github.com/Alpha-Labs-Global/blockalizer/blob/master/.github/workflows/ubiquibot-config.yml

Pls refactor to read from the bot's config, not from an environment variable

Reply:
That's my fault for not clarifying this in the bounty! But yes it is necessary to read from the yml file.

Reply:
> Pls refactor to read from the bot's config, not from an environment variable

The latest commit is a refactor like you asked, now available from the config object

Thank you